docker/Makefile

```diff
@@ -0,0 +1,12 @@
+.DEFAULT_GOAL := build
+
+IMAGE_NAME = kozea/radicale
```

Comment:
`:=`

Reply:
what would be the effect?

Reply:
Well, it's a minor nitpick, like the `.DEFAULT_GOAL`.

Reply:
so, `=` and `:=` are semantically identical and your nitpicking on the inconsistence?

Reply:
Yes.

Reply:
Yes.